ci-operator/config/openshift/kueue-operator/openshift-kueue-operator-main.yaml

```diff
@@ -139,6 +145,20 @@ tests:
     - chain: kueue-operator-post
     test:
     - chain: kueue-operator-test-e2e
+- as: test-e2e-4-19-hypershift
+  minimum_interval: 96h
+  presubmit: true
+  steps:
+    cluster_profile: aws
```

Comment:
can you with aws-2 like it's done here?
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/release/openshift-release-master__hypershift-4.19.yaml#L17
The name of the zone that is failing is defined here https://github.com/openshift/release/blob/master/ci-operator/step-registry/hypershift/hostedcluster/create/hostedcluster/hypershift-hostedcluster-create-hostedcluster-commands.sh#L41-L47